build/size-reporter.ts

```diff
@@ -1,68 +1,17 @@
 import chalk from 'chalk'
+import { FileSizeReporter } from 'rollup-plugin-filesize'
```

Comment:
Although for consistency I would request you to add `type` keyword in front of the importing member but since this is not public using file I will consider this minor.

Reply:
Yeah, next PR~🤪